.github/workflows/post-merge-tasks.yml

```diff
@@ -0,0 +1,51 @@
+name: Release Drafter
```

Comment:
Oh right, I missed that. I didn't expect several different jobs to be in one action.

Maybe the display name of the job should be changed accordingly.